packages/ui-library/src/components/input-field-text/index.css.ts

```diff
@@ -86,10 +86,11 @@ export const styleCustom = typeSafeNestedCss/*css*/ `
             }
           }
 
-          &.disabled {
+          &[disabled] {
             &::placeholder {
               color: ${inputfield.placeholder.textcolor.default.disabled};
             }
+            color: ${inputfield.placeholder.textcolor.default.disabled};
```

Comment:
we should probably remove this. see previous comment

Reply:
Hi @faselbaum 
Thanks for pointing out. I have added the input color for disabled state within .blr-input-wrapper .blr-form-input
and removed  it here